connaisseur/image.py

```diff
@@ -25,7 +25,13 @@ class Image:
     tag: str
     digest: str
 
-    def __init__(self, image: str):
+    index: int
+    container_type: str
+
+    def __init__(self, image: str, index: int, container_type: str):
```

Comment:
This doesn't really define an image though. The position of an image in a deployment should not be part of an image; maybe it would be better to define a wrapper class ContainerImage/DeploymentImage/AdmissionImageInfo(?) that holds `(image: Image, index: int, container_type: str)` instead of pushing it onto the image class. Then my comment above about `c_image in parent_images` also works again if you compare Images by their full-name, which would be what one expects

Reply:
I added the `Container` class as a wrapper.